qt/res/OrganicMaps.desktop

```diff
@@ -7,6 +7,6 @@ Comment[ru]=Подробная оффлайновая карта мира
 Icon=organicmaps
 TryExec=OMaps
 Exec=OMaps
-Categories=Maps;Education;Geography;Geoscience;Qt;
+Categories=Maps;Utility;Geography;Geoscience;Qt;
 Keywords=Map;Maps;Offline Maps;OMaps;Organic Maps;OrganicMaps;OSM;OpenStreetMap;
```

Comment:
Hmm semicolon-delimited `Keywords` field has been standardized only in 1.1 version of the spec, but this file claims to conform to 1.0.
https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.0.html#introduction
https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html#introduction

Reply:
While it is indeed optional, I think we should strive to comply with the latest `1.5`, which is already 4 years old.
BTW, I have added a minimal validation to the CI in https://github.com/organicmaps/organicmaps/pull/8255